lib/install/management/dispatcher.go

```diff
@@ -147,6 +149,31 @@ func (d *Dispatcher) initDiagnosticLogs(conf *metadata.VirtualContainerHostConfi
 	}
 }
 
+func (d *Dispatcher) registerExtension(conf *metadata.VirtualContainerHostConfigSpec, settings *InstallerData) error {
```

Comment:
note to self: change this to take the extension pointer instead of installerdata pointer and pass the struct at method call time instead, so that this method could be reused to register another extension